config.py

```diff
@@ -0,0 +1,41 @@
+import redis
+import os
+from pathlib import Path
+from dotenv import load_dotenv
+from cachelib.file import FileSystemCache
+
+load_dotenv()
+
+
+class Config(object):
+    ENV = os.getenv("PYDENTITY_WALLET_ENV", "development")
+    DEBUG = True if ENV == "development" else False
+    TESTING = True if ENV == "development" else False
+
+    DOMAIN = os.getenv("DOMAIN", "localhost")
+
+    APP_NAME = "PyDentity Wallet"
+    APP_URL = f"https://{DOMAIN}"
```

Comment:
Is `https` always going to be required? Wondering if this should allow for `http` to match local/development deployments?

Reply:
I believe PWA requires https for full functionality, this is a security feature implemented by browsers. I'm ok with making this a separate variable to be defined at deployment and give this warning.